build.gradle

```diff
@@ -240,7 +240,7 @@ project(':core') {
 
   dependencies {
     compile project(':clients')
-    compile project(':log4j-appender')
+    compile "$slf4jlog4j"
```

Comment:
This is unfortunate. Ideally, Zookeeper shouldn't specify explicitly dependency on slf4j-log4j. We can keep the changes you have here.